server/bleep/src/cache.rs

```diff
@@ -420,10 +433,16 @@ impl<'a> FileCache {
                 info!(
                     repo_name,
                     relative_path, new, updated, deleted, "Successful commit"
-                )
+                );
+                InsertStats {
```

Comment:
It may be worth turning the return value of `commit()` into `InsertStats`, so we don't have both the tuple and this.